LICENSE.md

```diff
@@ -93,21 +90,21 @@ A Derivative must contain changes which a reasonably informed person would consi
 The User may distribute a Derivative if the following conditions are met:
  - The Derivative does not generate revenue.
  - The Derivative provides credit to the Author.
- - The Derivative complies with the Mod license.
+ - The Derivative, as a whole, is licensed under this license for anyone who comes into possession of a copy.
```

Comment:
Far too convoluted. If you want to improve the wording of this, don't make it more complex. Stating that the Derivative must meet the condition of compliance with the Mod license is sufficient, as far as I can see. What's the reasoning behind making this more specific?